quickfort.lua

```diff
@@ -0,0 +1,203 @@
+-- DFHack-native implementation of the classic Quickfort utility (PRE-RELEASE)
+--[====[
+
+quickfort
+=========
+Processes Quickfort-style blueprint files. This is a pre-release -- not all
```

Comment:
Is this still accurate?

Reply:
Ha! good catch. no. the script is feature complete. I'll submit a mini PR to fix this